scripts/west_commands/bindesc.py

```diff
@@ -186,6 +193,15 @@ def custom_search(self, args):
         custom_tag = self.bindesc_gen_tag(custom_type, int(args.id, 16))
         self.common_search(args, custom_tag)
 
+    def get_offset(self, args):
+        image = self.get_image_data(args.file)
+
+        magic = struct.pack('>Q' if self.is_big_endian else 'Q', self.MAGIC)
+        index = image.find(magic)
```

Comment:
What if the magic value would occur multiple times?

Reply:
The magic is 8 bytes long, so it's very unlikely to appear randomly in the image. What about other appearances of the magic, e.g. two images in one file? Well, currently it's not handled. We could add `skip` or `start_offset` arguments, but that deserves more thought. I would say this is very unlikely to happen, as the descriptors are by design placed as close as possible to the start of the image.